portfolio/src/main/webapp/script.js

```diff
@@ -23,28 +23,54 @@ function hover(){
     }
     else{
-    	paragraphTitle = "" + info;
-    	console.log(paragraphTitle + " clicked")
-    	if (paragraphTitle == "about"){
-        	aboutMeContainer.innerText = "ABOUT Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.ABOUT "
-    	}
-		else if (paragraphTitle == "projects"){
-        	aboutMeContainer.innerText = "PROJECTS Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.PROJECTS "
-    	}
-		else if (paragraphTitle == "facts"){
-       	 	aboutMeContainer.innerText = "FACTS Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.FACTS "
-    	}
-    	else if (paragraphTitle == "links"){
-        	aboutMeContainer.innerText = "LINKS Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.LINKS "
-    	}
+        // Clears previously active tab, changes class of newly active tab
+        var alreadyOpenTab = document.getElementsByClassName("activeTab");
+        if (alreadyOpenTab.length > 0)
+            alreadyOpenTab[0].className = alreadyOpenTab[0].className.replace(" activeTab", "");
+    	paragraphTitle = info;
+        buttonClicked.className += " activeTab";
+        // Fills content for respective tab opened
+    	if (paragraphTitle == "about")
+        	aboutMeContainer.innerText =  aboutMe;
+		else if (paragraphTitle == "projects")
+        	aboutMeContainer.innerText = pastProjects;
+		else if (paragraphTitle == "facts")
+       	 	aboutMeContainer.innerText = funFacts;
```

Comment:
Always use brackets w/ if statements. Even though they're technically optional bets practice is to always use them. It makes it more clear whats happening and harder to shoot your self in the foot with formatting,